### PR TITLE
[CtsNews] Fix YouTube embed video url regex and test cases

### PR DESCRIPTION
**Pull request by chien-yu**
# Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Fixed the YouTube embedded video URL regex pattern, somehow it was missing the "https" part in the beginning. I have tested this extractor's 3 unit test cases and updated the other 2 description.

### CODE REVIEW

File: youtube_dl/extractor/ctsnews.py [hunk 1]
```diff
@@ -63,7 +63,8 @@ def _real_extract(self, url):
         else:
             self.to_screen('Not CTSPlayer video, trying Youtube...')
             youtube_url = self._search_regex(
-                r'src="(//www\.youtube\.com/embed/[^"]+)"', page, 'youtube url')
+                r'src="(https?://www.youtube.com/embed/[^"]+)"', page, 'youtube url')
```
Comment (dstftw) [inline youtube_dl/extractor/ctsnews.py]: No. _extract_urls of youtube extractor.

File: youtube_dl/extractor/ctsnews.py [hunk 2]
```diff
@@ -63,7 +63,8 @@ def _real_extract(self, url):
...
+            print(youtube_url)
```
Comment (dstftw) [inline youtube_dl/extractor/ctsnews.py]: Remove all garbage.

File: youtube_dl/extractor/ctsnews.py (hunk 1 above)
Comment (chien-yu) [reply youtube_dl/extractor/ctsnews.py]: Thanks, done.

File: youtube_dl/extractor/ctsnews.py (hunk 2 above)
Comment (chien-yu) [reply youtube_dl/extractor/ctsnews.py]: Removed.

**Pull request merged**